planner/core/explain.go

```diff
@@ -171,7 +171,9 @@ func (p *PhysicalTableScan) TP() string {
 
 // ExplainInfo implements Plan interface.
 func (p *PhysicalTableScan) ExplainInfo() string {
-	return p.AccessObject().String() + ", " + p.OperatorInfo(false)
+	var buffer strings.Builder
+	buffer.WriteString(p.AccessObject().String() + ", " + p.OperatorInfo(false))
+	return buffer.String()
```

Comment:
Because I changed the implementation of operatorinfo, I feel that it will be more beautiful to use stringbuilder. But it doesn't matter and you can leave it alone.